master/internal/command/command.go

```diff
@@ -126,7 +126,7 @@ func (c *command) Receive(ctx *actor.Context) error {
 
 		priority := c.Config.Resources.Priority
 		if priority != nil {
-			if err := c.setPriority(ctx, *priority); err != nil {
+			if err := c.setPriority(ctx, *priority, true); err != nil {
```

Comment:
erroring the actor here is right?

Reply:
yeah there is an error that should be raised in the event the RM is kubernetes

Reply:
chatted offline, i think this would mean you couldn't launch any commands with priority, so maybe we tweak this a bit to allow at least the initial set priority.